### PR TITLE
[Technical debt] LPS-172894 Migrate Object's Validations tab to React

### PR DESCRIPTION
**Pull request by alinedoleron**
[LPS-172894](https://issues.liferay.com/browse/LPS-172894) Migrate Object's Validations tab to React

Depends on: https://github.com/liferay-objects/liferay-portal/pull/2145

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:objects

**Comment by alinedoleron** [comment]
ci:test:object

**Comment by alinedoleron** [comment]
ci:test:object

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
Wait for https://github.com/liferay-objects/liferay-portal/pull/2145

**Review by naraandrade — CHANGES_REQUESTED** [review]
@alinedoleron Same comments as in https://github.com/liferay-objects/liferay-portal/pull/2153. Could you take a look? Thanks :)

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/Validation.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+} from '../../utils/fds';
+
+interface ItemData {
+	active: boolean;
+	id: number;
+}
+
+const language = Liferay.ThemeDisplay.getBCP47LanguageId();
+
+export default function Validation({
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/Validation.tsx:42]: @alinedoleron Please, change the file and component names to "Validations".

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:forward

**Comment by naraandrade** [comment]
ci:forward

**Comment by JhossephG** [comment]
ci:test:object

**Comment by naraandrade** [comment]
@alinedoleron Please, see this [thread](https://liferay.slack.com/archives/C048YNC5GRK/p1686574728275789).

**Comment by alinedoleron** [comment]
ci:test:object

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:forward

**Comment by alinedoleron** [comment]
ci:forward:force

**Comment by alinedoleron** [comment]
ci:forward:force

**Pull request closed without merging**